src/tools/createIssue.ts

```
@@ -259,16 +259,99 @@ export async function createIssue(args: any) {
-            errorMsg = `Error ${error.response.status}: ${JSON.stringify(error.response.data) || error.message}`;
-        } else if (error.message) {
-            errorMsg = error.message;
+        // Handle validation errors (from Yup schema)
+        if (error.name === "ValidationError") {
+            errorTitle = "Validation Error";
+            errorMsg = "The provided data does not meet the requirements for creating a Jira issue.";
+            errorDetails = error.message;
+            errorSolution = "Please check the field requirements and provide all necessary information.";
         }
+        // Handle authentication/credential errors
+        else if (error.message && error.message.includes("credentials")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)